modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/am/integration/test/impl/RestAPIPublisherImpl.java

```diff
@@ -668,6 +668,25 @@ public HttpResponse addContentDocument(String apiId, String docId, String docCon
         return response;
     }
 
+    /**
+     * Updating the document content using file
+     *
+     * @param apiId      - Id of the API.
+     * @param docId      - document Id.
+     * @param docContent - file content
+     * @return HttpResponse - Response  with Document adding result.
+     * @throws ApiException - Exception throws if error occurred when adding document.
+     */
+    public HttpResponse updateContentDocument(String apiId, String docId, File docContent) throws ApiException {
+        DocumentDTO doc = apiDocumentsApi.apisApiIdDocumentsDocumentIdContentPost(apiId, docId, docContent, null,
+                null);
+        HttpResponse response = null;
+        if (StringUtils.isNotEmpty(doc.getDocumentId())) {
+            response = new HttpResponse("Successfully update the documentation", 200);
```

Comment:
Shall we corrected the msg as "Successfully updated the documentation"

Reply:
Fixed with b518b9f